BOM/pom.xml

```diff
@@ -0,0 +1,105 @@
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.sap.cloud.security.xsuaa</groupId>
+    <artifactId>BOM</artifactId>
+    <version>2.11.3</version>
+    <packaging>pom</packaging>
+    <name>sap-cloud-security-BOM</name>
+
+    <url>https://github.com/SAP/cloud-security-xsuaa-integration</url>
+    <description>BOM of XSUAA authentication and authorization service integration libraries and samples
+    </description>
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>XSUAA Development</name>
+            <email>xsuaa-development@sap.com</email>
+            <organization>SAP SE</organization>
+            <organizationUrl>https://www.sap.com</organizationUrl>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:git://github.com/SAP/cloud-security-xsuaa-integration.git</connection>
+        <developerConnection>sscm:git:git://github.com/SAP/cloud-security-xsuaa-integration.git</developerConnection>
+        <url>http://github.com/SAP/cloud-security-xsuaa-integration/tree/master</url>
+    </scm>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+        <repository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+    </distributionManagement>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <parent-groupId>com.sap.cloud.security</parent-groupId>
+    </properties>
+
+    <dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>token-client</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>${parent-groupId}</groupId>
+            <artifactId>java-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>spring-xsuaa</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>spring-xsuaa-test</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>${parent-groupId}</groupId>
```

Comment:
pls double check all group Ids java-security, spring-security and some others have no xsuaa suffix....